<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindowClass</class>
 <widget class="QMainWindow" name="UserWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>648</width>
    <height>648</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>648</width>
    <height>648</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushSwitchUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>631</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>631</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>Switch to Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSearchButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="displayTextSearch">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBooksText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushYesAnswer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes, I want to add the book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNoContinueAnswer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>No, but I want to continue with the next book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNoEndAnswer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>No, but I want to finish this process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDisplayButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Display the Booklist</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loadNameFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushExitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Exit Application</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLoadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Load the Booklist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Save the Booklist</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserDisplay">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>390</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>231</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>231</height>
     </size>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="displayStatusSearch">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_time">
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
